### DISCUSSION TITLE
Deployment: commit or ignore content. Best approach?

### DISCUSSION THREAD

**Discussion by jannejava** [statamic/cms#5583] [General]
Hi!

I am curious to hear your experience regarding content and git? Committing the whole `content`-folder has its pros and cons. We are a team, and I am not updating the content. The server will not have write access to the git repo, so on every deployment, we have to sync before deploying. Or is there a more granular approach where you commit the YAML-files and ignore the content's markdown-files?  

On the other side, it's nice to be able to update collections with new fields without effort. 

What's your experience? I really appreciate any help you can provide.

**Comment by jasonvarga** [comment]
You can customize where your content files live.

https://statamic.dev/stache#stores

For instance, you could place your `entries` outside of the Git repo if you don't want to version them, but keep your `collections` in the repo so that the yaml files are versioned.

**Reply by duncanmcclean** [reply]
You could also choose to ignore any Markdown files in the `content` directory from being version controlled, using a `.gitignore`

**Comment by jannejava** [comment]
Thanks for your replies. Those are two good options!

I think I will try with the gitignore-option (why didn't I think of that?) since it's multisite.
